app/user/profile/ProfilePageLayout.tsx

```diff
@@ -20,14 +21,17 @@ const ProfilePageLayout = async () => {
   return (
     <div className="content-container grid-go -mt-space-y w-full space-y-3">
       <div className="col-span-full flex flex-row flex-wrap">
-        <h1 className="text-typo-subtitle-sm mb-5 lg:w-full">Profile</h1>
+        <h1 className="text-typo-subtitle-sm mb-5 opacity-50 lg:w-full">Profile</h1>
```

Comment:
This will also work and apply only on the text. Makes no different in this case though. :)

Remember to check darkmode when adding opacity. 50% looks very different with each background.

```suggestion
        <h1 className="text-typo-subtitle-sm mb-5 text-foreground/50 lg:w-full">Profile</h1>
```